models/news.models.js

```diff
@@ -8,7 +8,7 @@ exports.fetchTopics = () => {
 };
 
 exports.fetchArticleById = (article_id) => {
-    return db.query("SELECT * FROM articles WHERE article_id = $1;", [article_id]).then((article) => {
+    return db.query("SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.body, articles.created_at, articles.votes, COUNT(comments.article_id) AS comment_count FROM articles INNER JOIN comments ON articles.article_id = comments.article_id WHERE articles.article_id = $1 GROUP BY articles.article_id;", [article_id]).then((article) => {
```

Comment:
you could do SELECT * which would make this easier to read :)

Reply:
Also, have a look if there is a way in your query to make the comment count column a number rather than a string which is what it will currently be